setup.py

```diff
@@ -22,6 +22,7 @@
         "pytz>=2016.10",
         "shortuuid>=0.4.3",
         "tabulate>=0.7.7",
+        "pathlib2; python_version < '3.4'",
```

Comment:
Okay, turns out specifying the python_version requires a newer version of setuptools during install time as well. I don't think we should put that requirement to the user. Would it be fine if we install pathlib2 in all envs? Seems like it does not break things in python3.